app/lib/finder_api.rb

```diff
@@ -46,6 +46,12 @@ def augment_content_item_with_results(content_item, search_response)
     content_item.details.results = search_response.fetch("results")
     content_item.details.total_result_count = search_response.fetch("total")
 
+    content_item.details.pagination = build_pagination(
+      content_item.details.default_documents_per_page,
+      search_response.fetch('start'),
+      search_response.fetch('total')
+    )
```

Comment:
I was wondering if instead of adding the pagination to the content_item's details, the FinderApi could return the content_item and the pagination data as a kinda `return [content_item, pagination]`... But I haven't worked with finders much, so maybe the pagination in the `details` already makes enough sense.